### PR TITLE
#1330: make cbs port retries configurable

### LINKED ISSUE #1330: Make the maximum number of CBS port retries configurable

### PR DESCRIPTION
**Pull request by guangningyu**
Fixes #1330

### CODE REVIEW

File: src/main/python/smv/smvapp.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def __init__(self, arglist, _sparkSession = None):
 
         # check wither the port is in-use or not. Try 10 times, if all fail, error out
```
Comment (cloudysail) [inline src/main/python/smv/smvapp.py]: need to update comment

**Comment by AliTajeldin** [comment]
Need to update the user guide as well to document the new configuration parameter.

**Comment by guangningyu** [comment]
@AliTajeldin @cloudysail @laneb 
I've made the changes, please have a look again.

**Comment by AliTajeldin** [comment]
Needs approval from @cloudysail as he had feedback.

**Pull request merged**